tests/NCronJob.Tests/TestHelper.cs

```diff
@@ -189,3 +193,23 @@ private async IAsyncEnumerable<object> GetCompletionJobsAsync(int expectedJobCou
         }
     }
 }
+
+public sealed class Storage
+{
+#if NET9_0_OR_GREATER
+        private readonly Lock locker = new();
+#else
+    private readonly object locker = new();
+#endif
+#pragma warning disable CA1002 // Do not expose generic lists
+    public List<string> Entries { get; private set; } = [];
```

Comment:
Instead of the `pragma disable` - can't it be just a `IReadOnlyList`?

Reply:
We need to able able to leverage `List<T>` indexers to ensuring ordering of entries when possible.

Reply:
Yes - `IReadOnlyList` would stll allow for such scenario, wouldn't it?

Reply:
I am just not a big fan of having those pragmas around. Either the rule doesn't make sense for tests, then we should exclude our tests from those rules.

Or they make somewhat sense and we can "fix" them. 

As `Entries` is anyway used only in "reading" - `IReadOnlyList` would fix the pragma and still would allow everything you need. Or maybe I am missing something here? @nulltoken